lib/twine/cli.rb

```diff
@@ -89,6 +89,12 @@ def self.parse(args)
           end
           options[:output_encoding] = e
         end
+        opts.on('-b', '--deliberate', 'Validate the strings file before formatting it') do
```

Comment:
I'm fine with any other letter that isn't being used that is also in the word "validate". :)